meinberlin/assets/scss/components/_a4-comments.scss

```diff
@@ -30,19 +30,10 @@
     }
 
     .row {
-        @extend .l-flex-row;
```

Comment:
removed extends from user facing components as these will be redone with BO classes